docs/src/components/FilterChildren.tsx

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from 'next/router';
+
+export const FilterChildren = ({ children, filterKey = '' }) => {
```

Comment:
Do we need 2 separate components? Could we just have a single component? I don't think this one would ever be used on its own right?

Reply:
Nope 2 separate components are not needed for this, they have been combined and simplified into a single component.